### PR TITLE
[MSPAINT] Update Turkish (tr-TR) translation

### PR DESCRIPTION
**Pull request by mahirgul**
Added translations.

## Purpose

_Do a quick recap of your work here._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

File: base/applications/mspaint/lang/tr-TR.rc [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * PURPOSE:     Turkish resource file
  * TRANSLATORS: Copyright 2013-2016, 2019 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
  *              Copyright 2021-2022 Süleyman Poyraz <zaryob.dev@gmail.com>
+ *              Copyright 2024 Mahir GÜL <mahirgul@gmail.com>
```
Comment (JoachimHenze) [inline base/applications/mspaint/lang/tr-TR.rc:7]: Are you sure, that it must be GÜL here and not Gül ?

**Review by JoachimHenze — APPROVED** [review]
Approved. (and selected for insta-merge when the bots will have ack'ed)

**Comment by binarymaster** [comment]
@JoachimHenze there's existing tr-TR pull request, maybe it should be processed first?

https://github.com/reactos/reactos/pull/7291

**Comment by binarymaster** [comment]
Also this PR is not yet reviewed by @reactos/lang-turkish

**Comment by JoachimHenze** [comment]
> @JoachimHenze there's existing tr-TR pull request, maybe it should be processed first? #7291

7291 is not ready to be merged. But this one here seems to be much more polished already. So I don't see why this one here should have to wait for the other one. If there would evolve a need for resolving conflicts, that is similar effort for both, no matter which one will be merged first.

**Pull request merged**